uber/site_sections/emails.py

```diff
@@ -71,3 +71,25 @@ def test_email(self, session, subject=None, body=None, from_address=None, to_add
     def approve(self, session, subject):
         session.add(ApprovedEmail(subject=subject))
         raise HTTPRedirect('pending?message={}', 'Email approved and will be sent out shortly')
+
+    def emails_by_interest(self):
+        return {}
+
+    @csv_file
+    def emails_by_interest_csv(self, out, session, *args, **kargs):
+        """
+        Generate a list of emails of attendees who match one of c.INTEREST_OPTS
+        (interests are like "LAN", "music", "gameroom", etc.
```

Comment:
The lack of a closing parenthesis here is REALLY bothering me, way out of proportion to how much it reasonably should bother me :(